backend/package-lock.json

```diff
@@ -0,0 +1,7845 @@
+{
```

Comment:
- use yarn and remove package-lock
- fix linter warning - see https://github.com/Yaroslaww-1/cte/pull/18/checks?check_run_id=2143654762